manifest.json

```diff
@@ -3157,6 +3386,614 @@
+                    "artifacts": [
+                        {
+                            "url": "https://github.com/badhaloninja/UserJoinLeaveNotifications/releases/download/v1.1.1/UserJoinLeaveNotifications.dll",
+                            "filename": "UserJoinLeaveNotifications.dll",
+                            "sha256": "884678e0cdc18cda7ddde79ff88fda6b9069ecf75e63a8b1504b67ec1fb6ba4c"
+                        }
+                    ]
+                },
+                "1.1.0": {
+                    "changelog": "Fixed issue of join leave notifications only showing if you are host",
+                    "releaseUrl": "https://github.com/badhaloninja/UserJoinLeaveNotifications/releases/tag/v1.1.0",
+                    "artifacts": [
+                        {
+                            "url": "https://github.com/badhaloninja/UserJoinLeaveNotifications/releases/download/v1.1.0/UserJoinLeaveNotifications.dll",
+                            "filename": "UserJoinLeaveNotifications.dll",
+                            "sha256": "be07fdb9fee384ae2162ed759050e53952ef28a5f21ebcc532691d8f98ad842a"
+                        }
+                    ]
+                },
+                "1.0.0": {
+                    "releaseUrl": "https://github.com/badhaloninja/UserJoinLeaveNotifications/releases/tag/v1.0.0",
+                    "artifacts": [
+                        {
+                            "url": "https://github.com/badhaloninja/UserJoinLeaveNotifications/releases/download/v1.0.0/UserJoinLeaveNotifications.dll",
+                            "filename": "UserJoinLeaveNotifications.dll",
+                            "sha256": "3da38886a64878fc9fffb674b93e86ea95ec1fe8344302d6aa70ade9c163c854"
+                        }
+                    ]
+                }
+            }
+        },
+        "me.badhaloninja.DesktopNotifications": {
+            "name": "DesktopNotifications",
+            "description": "Shows notifications in desktop mode",
+            "category": "Dash Tweaks",
+            "sourceLocation": "https://github.com/badhaloninja/DesktopNotifications",
+            "authors": {
+                "badhaloninja": {
+                    "url": "https://github.com/badhaloninja"
+                }
+            },
+            "versions": {
+                "1.1.0": {
+                    "changelog": "Switch to using float2 instead of 2 separate float options",
+                    "releaseUrl": "https://github.com/badhaloninja/DesktopNotifications/releases/tag/v1.1.0",
+                    "artifacts": [
+                        {
+                            "url": "https://github.com/badhaloninja/DesktopNotifications/releases/download/v1.1.0/DesktopNotifications.dll",
+                            "filename": "DesktopNotifications.dll",
+                            "sha256": "547426ba0ff8f02a15bd63031838d2ace58b5c65e5b0085096e0a09df20e9ce6"
+                        }
+                    ]
+                },
```

Comment:
IL transpiler stuff though looks fine as it only ever removes a code instruction.